dex/elrond_dex_router/src/factory.rs

```diff
@@ -166,28 +147,31 @@ pub trait FactoryModule {
         }
     }
 
+    #[only_owner]
     #[endpoint(clearPairTemporaryOwnerStorage)]
     fn clear_pair_temporary_owner_storage(&self) -> SCResult<usize> {
```

Comment:
Remove SCResult<()> and return nothing is you have no additional checks. #[only_owner] handles the error "outside" of the function body, so there's no need to manually return Ok(()).  Simply return `usize`

Reply:
done